src/lib/holocene/pagination.svelte

```diff
@@ -91,7 +86,8 @@
           <Icon name="chevron-left" />
         </button>
         <p>
-          {$store.startingIndex + 1}–{$store.endingIndex + 1} of {$store.length}
+          {$store.length ? $store.startingIndex + 1 : 0}–{$store.endingIndex +
```

Comment:
It looks like the total workflows and the pagination total don't always match 😕 

![Screen Shot 2022-10-12 at 9 40 17 AM](https://user-images.githubusercontent.com/15069288/195396020-9d69d9ae-499c-4f62-9ae6-dd221920d670.png)